CONTRIBUTING.md

```diff
@@ -8,13 +8,37 @@ Glad that you decided to make your contribution in Dash, to set up your developm
 # in your working directory
 $ git clone https://github.com/plotly/dash
 $ cd dash
-# create a virtualenv
-$ python3 -m venv venv
-# activate the virtualenv (on windows venv\scripts\activate)
-$ . venv/bin/activate
-# Install the dev dependencies
-$ pip install -r .circleci/requirements/dev-requirements.txt
+$ python3 -m venv .venv/dev
+# activate the virtualenv (on windows .venv\dev\scripts\activate)
+$ . .venv/dev/bin/activate
+# install dash and dependencies
+$ pip install -e .[testing,ci]  # might need \ to espcae []
```

Comment:
~escpae~ escape

But, is that true? Is this a concern on a particular OS or something?

Reply:
on zsh, you need \ to escape

Reply:
OK, so let's change to 
```suggestion
$ pip install -e .[testing,ci]  # in some shells you need \ to escape []
```